dashboard/config/scripts/levels/Allthethings Java Lab 1.level

```diff
@@ -4,13 +4,20 @@
     "encrypted": "false",
     "instructions_important": "false",
     "teacher_markdown": "Teachers should see this extra info.",
-    "long_instructions": "This is a sample Java Lab level."
+    "long_instructions": "This is a sample Java Lab level.",
```

Comment:
just a sanity-check -- you want to update this level as part of your PR, right? (i ask because i've done this accidentally... many times)

Reply:
Haha yes. Thank you for the sanity check :)